DESCRIPTION

```diff
@@ -9,7 +9,7 @@ Description: This is a customized fork of the original work from Tianwei Yu.
         It takes the adaptive processing of LC/MS metabolomics data further
         with focus on high resolution MS for both LC and GC applications.
 Depends: R (>= 3.50), MASS, mzR, splines, doParallel, foreach,
-        snow, dplyr, tidyr, stringr, tibble, tools, arrow
+        snow, dplyr, tidyr, stringr, tibble, tools, arrow, microbenchmark
```

Comment:
```suggestion
        snow, dplyr, tidyr, stringr, tibble, tools, arrow
```

Reply:
This should go to suggests

Reply:
done in [563b98a](https://github.com/RECETOX/recetox-aplcms/pull/131/commits/563b98a5e5d9a4619f18d37ae10f411e4d0f58c5)